en/docs/authenticate/saml-app-config-advanced.md

```diff
@@ -0,0 +1,218 @@
+## Advanced SAML Configurations 
+
+----
+
+#### Issuer 
+
+This is the `<saml:Issuer>` element that contains the unique identifier of the service provider. This is also the issuer value specified in the SAML Authentication Request issued by the service provider.
+
+----
+
+#### Service Provider Qualifier
+
+This value is needed only if you have to configure multiple SAML SSO inbound authentication configurations for the same Issuer value. When a Service Provider Qualifier is defined here, it will be appended to the end of the **Issuer** value when registering the SAML SP in the Identity Server.
+
+> For example, if you specify `saml-pickup-dispatch.com` as the Issuer and `sp1` as the Service Provider Qualifier, the configuration will be registered in IS as `saml-pickup-dispatch.com:urn:sp:qualifier:sp1`.
+
+You can configure a number of SAML SPs with the same Issuer and different Service Provider Qualifiers.
+
+When a Service Provider Qualifier is defined, the issuer of the SAML SSO authentication request is the value specified as the Issuer in the configuration (e.g., `saml-pickup-dispatch.com`). The service provider qualifier value should be sent as a query parameter, `spQualifier` with the HTTP request in the following format.
+
+```
+https://{Hostname}:{Port}/samlsso?spQualifier={Service Provider Qualifier}
```

Comment:
Shall we use the same variable to represent the hostname. In some places, it is written as  'Hostname ' and in some places, it is written as 'host-name'.